rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -9,12 +9,7 @@
 import static org.eclipse.microprofile.lra.annotation.ws.rs.LRA.LRA_HTTP_CONTEXT_HEADER;
 import static org.eclipse.microprofile.lra.annotation.ws.rs.LRA.LRA_HTTP_RECOVERY_HEADER;
 import static org.hamcrest.Matchers.containsString;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
Please can I ask if it was intentionakl to remove these imports in this commit and replace with a .*? I can see you have a separate commit that follows up to remove the .* that is added in this one? https://github.com/jbosstm/narayana/pull/2312/commits/4220b84f76bc2c80fdf69c4cea22ca9003d44c87#diff-709c5ef85ed562bd70894c9781aaaae149a18d85eddc6f7d32902e1c387e0311L12

That commit also changes a lot of `assert[...]` to `Assert.assert[...]` which I don't quite understand? Is it something to move forwards with JUnit versions somehow?

Reply:
Look at the second commit where I organise the imports - I thought that was better than updating my settings (which I'll do anyway) but I want to discuss the issue with Marco.

Reply:
Yeah, I had seen the other commit but I am still not clear on why the need to change from (e.g.) assertEquals to Assert.assertEquals (etc)?

Reply:
I think the only advantage to using the static calls is to improve readability of the source files. I guess it is a matter of developer preference, but we can avoid using static Asserts if that is what the team prefers.

Reply:
I also notice that the hudson script sets `ARQ_PROF=no_arq` - I've not seen that before but it doesn't exist so I'll need to investigate that too.

Reply:
> I think the only advantage to using the static calls is to improve readability of the source files. I guess it is a matter of developer preference, but we can avoid using static Asserts if that is what the team prefers.

I'll remove them and if the team wants to allow it then we discuss that separately.

Reply:
Thanks for your question Mike, I added the 'no_arq' profile for JDK 11 because we cannot test Wildfly with that JDK only to avoid the arq profile to be enable. I can see now that there is the warning. Maybe it would be better to set ARQ_PROF="" ?

Reply:
If ARQ_PROF isn't valid for the JDK11 tests then unsetting ARQ_PROF is best: `unset ARQ_PROF`